File: testprojects/master_python_mainone/pipelines/StreamingAllPipeline/code/streamingallpipeline/streaming/main/one/this/graph/SchemaTransform_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from streamingallpipeline.streaming.main.one.this.config.ConfigStore import *
from streamingallpipeline.streaming.main.one.this.udfs.UDFs import *

def SchemaTransform_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.withColumn("c_new_col", concat(col("c_boolean"), col("c_float")))

File: testprojects/master_scala_mainone/pipelines/SCALA_DEPENDENCY_MANAGEMENT/code/src/main/scala/graph/SchemaTransform_1.scala
package graph

import io.prophecy.libs._
import udfs.UDFs._
import config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object SchemaTransform_1 {

  def apply(context: Context, in: DataFrame): DataFrame = {
    val Config = context.config
    in.withColumn("c_concat_new_short_decimal", expr(Config.c_st_expr))
      .drop(Config.c_st_long)
      .withColumnRenamed("c_array_boolean", Config.c_st_rename)
  }

}
